vars/docOnlyChange.groovy

```diff
@@ -14,14 +14,26 @@ boolean call(String target_branch) {
     if (cachedCommitPragma('Doc-only').toLowerCase() == 'true') {
         return true
     }
-    if (cachedCommitPragma('Doc-only').toLowerCase() == 'false' ||
-        !fileExists('ci/doc_only_change.sh')) {
+    if (cachedCommitPragma('Doc-only').toLowerCase() == 'false') {
         return false
     }
-
-    return sh(label: "Determine if doc-only change",
-              script: ["CHANGE_ID=${env.CHANGE_ID}",
-                       "TARGET_BRANCH=${target_branch}",
-                       'ci/doc_only_change.sh'].join(' '),
+    if (fileExists('ci/doc_only_change.sh')) {
+        script = 'CHANGE_ID= ' + env.CHANGE_ID +
+                 ' TARGET_BRANCH= ' + target_branch +
```

Comment:
You need to remove the spaces after the `=` in this command line build up.

```suggestion
        script = 'CHANGE_ID=' + env.CHANGE_ID +
                 ' TARGET_BRANCH=' + target_branch +
```

Reply:
Done